build.sh

```diff
@@ -65,7 +65,7 @@ case "$TRAVIS_OS_NAME" in
     npm install -g --save-dev electron-installer-redhat --silent
 
     # Install libgconf2-4
-    sudo apt-get install libgconf2-4
+    sudo apt-get install libgconf2-4 libgconf-2-4 
```

Comment:
Should this have both `libgconf2-4` and  `libgconf-2-4`?

Reply:
One is a shim and the other is real